contracts/Migrator.sol

```diff
@@ -5,26 +5,33 @@ import { ERC20Helper } from "../modules/erc20-helper/src/ERC20Helper.sol";
 
 import { IERC20Like } from "./interfaces/Interfaces.sol";
 
-contract Migrator {
+import { IMigrator } from "./interfaces/IMigrator.sol";
 
-    address public immutable newToken;
-    address public immutable oldToken;
+contract Migrator is IMigrator {
 
-    constructor(address oldToken_, address newToken_) {
+    uint256 public immutable override TOKEN_SPLIT_SCALAR;
```

Comment:
nit if not constant we should lowercase and order based on type i.e `uint256` after `address`